api/FileReaderSync.json

```
@@ -249,103 +249,59 @@
           }
         }
       },
-      "service_workers_support": {
+      "worker_support": {
         "__compat": {
-          "description": "Service workers support",
+          "description": "Available in workers (except for service workers)",
           "support": {
             "chrome": {
               "version_added": true,
-              "version_removed": "59"
+              "notes": "Not supported in service workers from Chrome 59"
             },
             "chrome_android": {
               "version_added": true,
-              "version_removed": "59"
+              "notes": "Not supported in service workers from Chrome 59"
             },
             "edge": {
-              "version_added": null
+              "version_added": "12",
+              "notes": "Not supported in service workers"
             },
             "firefox": {
-              "version_added": true,
-              "version_removed": "61"
+              "version_added": "8",
+              "notes": "Not supported in service workers from FF61"
             },
             "firefox_android": {
-              "version_added": true,
-              "version_removed": "61"
+              "version_added": "8",
+              "notes": "Not supported in service workers from FF61"
             },
             "ie": {
-              "version_added": false
+              "version_added": true,
+              "notes": "Not supported in service workers"
```

Maybe we should omit this note? If I remember correctly, IE doesn't support service workers in any case (e.g., doesn't implement `navigator.serviceWorker`)

```suggestion
              "version_added": true
```